### PR TITLE
Added Geeetech i3 files

### PR DESCRIPTION
**Pull request by martin-key**
I have added and tested Geeetech i3 configuration. 
Please review and comment.

### CODE REVIEW

File: Marlin/pins_GT2560.h [hunk 1]
```diff
@@ -0,0 +1,99 @@
+
```
Comment (thinkyhead) [inline Marlin/pins_GT2560.h:1]: This board is completely identical to `BOARD_ULTIMAKER`. If it's not a unique board, then you should just use this in `Configuration.h`:

``` cpp
#define MOTHERBOARD BOARD_ULTIMAKER
```

Or, if this is a unique board which happens to use all the same pin assignments as `BOARD_ULTIMAKER`, then this file should contain only the following:

``` cpp
/**
 * GT2560 pin assignments
 */
#include "pins_ULTIMAKER.h"

#undef DEFAULT_MACHINE_NAME
#define DEFAULT_MACHINE_NAME    "GT2560"
#undef DEFAULT_SOURCE_CODE_URL
```

File: Marlin/pins.h [hunk 2]
```diff
@@ -153,6 +153,8 @@
   #include "pins_MKS_13.h"
 #elif MB(SAINSMART_2IN1)
   #include "pins_SAINSMART_2IN1.h"
+#elif MB(GT2560)
```
Comment (thinkyhead) [inline Marlin/pins.h:156]: You will also need to define `BOARD_GT2560` in the `boards.h` file.

File: Marlin/pins_GT2560.h (hunk 1 above)
Comment (martin-key) [inline Marlin/pins_GT2560.h:1]: I think that the pinout it not the same. Must double check, but if it was my printer should have worked the first time. I can send you the schematic if you want to check. Thanks !

File: Marlin/pins.h (hunk 2 above)
Comment (martin-key) [inline Marlin/pins.h:156]: Yes I have not done that. Sorry !

File: Marlin/pins_GT2560.h (hunk 1 above)
Comment (thinkyhead) [inline Marlin/pins_GT2560.h:1]: I did a pin-by-pin comparison and found no differences. You can do the same, easily enough. Most pins defined as `-1` should now be left out of pins files, so some of the pins defined as `-1` in `pins_GT2560.h` are not present in `pins_ULTIMAKER.h`.

**Comment by thinkyhead** [comment]
You will also need to close this PR and create a new one targeted to the `RCBugFix` branch. You will probably need to create a new branch based on `RCBugFix` and start from there. It's best to make a new branch, and not to just modify your copy of `RCBugFix`. From the command line you would do this:

```
git remote add upstream git@github.com:MarlinFirmware/Marlin.git ;# ...or...
git remote add upstream https://github.com/MarlinFirmware/Marlin.git ;# with no ssh key
git fetch upstream
git checkout upstream/RCBugFix -b my_geeetech_support_branch
```

Then make your modifications, add your pins file, etc., and…

```
git add *
git commit -m "Geeetech i3 Support"
git push --set-upstream origin my_geeetech_support_branch
open https://github.com/MarlinFirmware/Marlin/compare/RCBugFix...StarbuckBG:my_geeetech_support_branch?expand=1
```

**Comment by martin-key** [comment]
OK thanks !

**Pull request closed without merging**